### DISCUSSION TITLE
About .mode files

### DISCUSSION THREAD

**Discussion by Saivellampati** [ansys/pydpf-core#1713] [Q&A] [answered]
Are we able to read .mode files alone without rst. I tried to read it, but it is giving error msg

```
ResultInfoProvider:4<-mapdl::mode::ResultInfoProvider:5<-GenPass file does not contains the mesh, and has no side rst to read it !
```
If add **.mode** file along rst, what kind of extra information we will get ?

Can someone help me in this ?

**Comment by rlagha** [comment]
Hi @Saivellampati , yes you can read data with only mode files, what kind of info you are looking for?

**Comment by Saivellampati** [comment]
@rlagha

I want mesh info and result info.  I am unable extract any of it. For your reference, I am attaching the file
[Uploading mode.zip…]()

**Comment by rlagha** [comment]
@Saivellampati , the mesh and result info are in the associated rst file. There is always an rst file generated next to the mode file. Do you have that?

**Comment by Saivellampati** [comment]
@rlagha 
yes I have that and i am able to get those information from the file. But then what is the use of .mode file. 
What kind of extra information we will get from it ?

**Comment by rlagha** [comment, marked as answer]
@Saivellampati 
you can find the content of the mode file in the MAPDL doc bellow:

https://ansyshelp.ansys.com/account/secured?returnurl=/Views/Secured/corp/v242/en/ans_prog/Hlp_P_INT1_5.html%23d0e601

**Comment by mikerife** [comment]
Hi @Saivellampati PyDPF accessing the mode file is mainly for retrieving the participation factors.  And I suppose since the file is being accessed and the frequency results are right there the eigenmodes as well.  Mike
